PathPlanning/GridBasedSweepCPP/grid_based_sweep_coverage_path_planner.py

```diff
@@ -249,13 +258,14 @@ def planning(ox, oy, resolution,
              ):
     sweep_vec, sweep_start_position = find_sweep_direction_and_start_position(
         ox, oy)
+    print("sweep_vec", sweep_vec, sweep_start_position)
```

Comment:
Why do you want to change this? It looks unrelated to import problem.

Reply:
This was my mistake. I forgot to remove some line that i tested. Sorry for inconvenience.